docs/release-notes/self-hosted/self-hosted-v4.1.0.md

```diff
@@ -0,0 +1,74 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v4.1.0.
+---
+
+# Self-hosted v4.1.0
+
+These release notes are for [Codacy Self-hosted v4.1.0](https://github.com/codacy/chart/releases/tag/4.1.0){: target="_blank"}, released on July 6, 2021. <!-- TODO Update release date -->
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+## Product enhancements
+
+-   The GitLab integration now supports [merge request summaries](https://docs.codacy.com/v4.1/repositories-configure/integrations/gitlab-integration/#enabling) for both GitLab Cloud and GitLab Enterprise. (CY-4310)
+-   You can now use markdownlint to provide [suggested fixes](https://docs.codacy.com/v4.1/repositories-configure/integrations/github-integration/#suggest-fixes) for Markdown issues. (CY-3717)
+
+## Bug fixes
+
+-   Fixed an issue that prevented Codacy from listing all available GitLab groups on the **Organizations** page. (CY-4529)
+-   Fixed an issue that caused Codacy to suggest and allow people with an empty email address to join organizations. (CY-4454)
+-   Fixed an issue that could allow malicious websites to read sensitive information or perform privileged actions on Codacy. CVSS v3.1 score: 3.3 (Low). (CY-4396)
+-   Fixed an issue where Codacy would show "Enable all" for non-configurable client-side tools on the **Code patterns** page. (CY-4323)
+-   Now, the link "View more details on Codacy Production" at the bottom of GitHub checks opens the **New Issues** tab instead of the **Fixed Issues** tab of the corresponding pull request on Codacy. (CY-4170)
```

Comment:
I know this was the way it was reported. But it wasn't only in the use case of "using link from provider side" it was in any scenario that includes a user visiting the PR/Commit page. We were opening the wrong tab by default. 

So I would suggest say something like 

```suggestion
-   When reaching the commit or pull request view we will open **New issues** tab instead of **Fixed Issues**. (CY-4170)
```

For more details, the whole condition is:
- By default we always open New Issues tab
- If we don't have new issues but have fixed issues then we open Fixed issues tab

Reply:
I don't know if we should go into this detail or not in the release notes, if not just ignore ;)

Reply:
It's nice that a wider scenario was improved. :+1: I think that like this, we're able to simplify a bit further:

```suggestion
-   Now, Codacy shows the **New Issues** tab by default when opening a commit or pull request page. (CY-4170)
```

What do you think?